src/test/java/api/InstancesApiExamples.java

```diff
@@ -187,6 +189,107 @@ public void canCreateAnInstanceWithAnID()
     selfLinkShouldBeReachable(createdInstance);
   }
 
+  @Test
```

Comment:
I think these tests could benefit from some whitespace. Maybe separating out the setup of each record, the batch submission and the checks on the response into groups.

Reply:
Done